_posts/The importance_of_being_earnest.md

```diff
@@ -0,0 +1,262 @@
+---
+title: The Importance Of Being Earnest
+
+author: Aditi A Pujar, Divyoj Singh, Bhoomika A Bhat, Akshara Sharma, R Bangari
```

Comment:
authors are to be identified using "tags" which we define in the config file (not sure if this will even work)

Reply:
Yeh, we will have to fix it sometime later, currently ill just put authors in the content itself, under the heading